### PR TITLE
[Bug-Fix] Minor change in Managed Network Fabric

### PR DESCRIPTION
**Pull request by Mohana-Krishna-Nali**
# ARM (Control Plane) API Specification Update Pull Request 

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

diagram

## Purpose of this PR

What's the purpose of this PR? Check all that apply. This is **mandatory**!

  - [ ] New API version. (Such PR should have been generated with [OpenAPI Hub](https://aka.ms/openapihub), per [this wiki doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).)
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix swagger quality issues in S360.
  - [X] Other, please clarify:
     
     ### Customer Requirement - Adding an optional param in couple of resources.
            
    New Params:

        1. In RoutePolicies: -> defaultAction [Permit | Deny -> default Deny] is added.
        2. In AccessControlLists: -> defaultAction [Permit | Deny -> default Permit] is added.

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [X] I have reviewed the general guidance on the spec PR review process: https://aka.ms/specprreview.
- [X] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
- [X] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
- [X] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can request review from an ARM API Review board.

### ARM API changes review

- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
- The automation may automatically add the `ARMReview` label, if appropriate.  
  If this happens, proceed according to guidance given in GitHub comments also added by the automation.

### Breaking change review

If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), 
follow the process outlined in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process).
      
## Getting help

- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For additional help, see https://aka.ms/azsdk/support/spectools.

### CODE REVIEW

**Review by abhishek-msft — APPROVED** [review]

File: specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/AccessControlLists.json [hunk 1]
```diff
@@ -645,6 +645,12 @@
           "format": "uri",
           "minLength": 1
         },
+        "defaultAction": {
+          "type": "string",
```
Comment (visingla-ms) [inline specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/AccessControlLists.json:649]: >string [](http://example.com/codeflow?start=19&length=6)

If the set of values are pre-defined, make it an enum?

Comment (Mohana-Krishna-Nali) [reply specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/AccessControlLists.json:649]: I have referred Enum, **CommunityActionTypes**

File: specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/RoutePolicies.json [hunk 2]
```diff
@@ -619,6 +619,12 @@
       "type": "object",
       "description": "Route Policy patchable properties.",
       "properties": {
+        "defaultAction": {
+          "type": "string",
```
[the remark this answers is not in the document]
Comment (Mohana-Krishna-Nali) [reply specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/stable/2023-06-15/RoutePolicies.json:623]: I have referred Enum, **CommunityActionTypes**

**Comment by Mohana-Krishna-Nali** [comment]
JFYI, We are trying to update the main with [Pull-Request](https://github.com/Azure/azure-rest-api-specs-pr/pull/13845), that is already got ARM-Signed off and merged in RPSaaSMaster.|


@visingla-ms /@rkmanda Could you please help review and merge??

**Comment by Mohana-Krishna-Nali** [comment]
@rkmanda, could you please help merge the PR?

**Review by rkmanda — APPROVED** [review]

**Pull request merged**